src/components/icon-search/parts/search-categories.css

```
@@ -0,0 +1,66 @@
+.p-category-select {
+	/* Layout */
+	display: inline-flex;
+
+	/* Layout */
+	align-items: center;
+
+	/* Layout */
+	margin-inline-end: 4--step;
+
+	/* Layout */
+	position: relative;
+
+	/* Layout */
+	width: fit-content;
+
+	/* Text */
+	font-family: var(--SystemType);
+	font-size: 9--step;
+
+	& select {
+		/* Layout */
+		margin-inline-end: 2--step;
+
+		/* Layout */
+		margin-inline-start: calc(-1 * 2--step);
+
+		/* Layout */
+		max-width: 100--step;
+		padding-block: 2--step;
+		padding-inline: 2--step;
+		width: 50--step;
+
+		/* Text */
+		font-size: 16--rpx;
+		line-height: 20--rpx;
+
+		/* Arrow */
+		appearance: none;
+
+		/* Appearance */
+		background-color: var(--PrimaryColor);
+		background-image: url("data:image/svg+xml;charset=utf8,%3Csvg xmlns='http://www.w3.org/2000/svg' viewBox='0 0 24 24'%3E%3Ctitle%3EKeyboard Arrow Down%3C/title%3E%3Cpath d='M8.12 9.29 12 13.17l3.88-3.88a.996.996 0 1 1 1.41 1.41l-4.59 4.59a.996.996 0 0 1-1.41 0L6.7 10.7a.996.996 0 0 1 0-1.41c.39-.38 1.03-.39 1.42 0Z'%3E%3C/path%3E%3C/svg%3E");
```

Nit: It may improve clarity to unescape some parts of this:

```suggestion
		background-image: url('data:image/svg+xml;charset=utf8,<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="M8.12 9.29 12 13.17l3.88-3.88a.996.996 0 1 1 1.41 1.41l-4.59 4.59a.996.996 0 0 1-1.41 0L6.7 10.7a.996.996 0 0 1 0-1.41c.39-.38 1.03-.39 1.42 0Z" /></svg>');
```